pkg/runner/flags.go

```diff
@@ -35,6 +35,10 @@ type Config struct {
 	InstallExtensions           bool
 	UpgradeExtensions           bool
 	UpgradePrereleaseExtensions bool
+	// Multi-tenancy.
+	EnableMultiTenancy  bool
+	ValidTenantsListStr string
```

Comment:
The string should never be used by anything
```suggestion
	validTenantsListStr string
```